patch-package
--- a/node_modules/@atlaskit/feedback-collector/dist/cjs/components/FeedbackForm.js
+++ b/node_modules/@atlaskit/feedback-collector/dist/cjs/components/FeedbackForm.js
@@ -171,14 +171,6 @@ function (_Component) {
             canBeContacted: event.target.checked
           });
         }
-      })), _react.default.createElement(_form.Field, null, _react.default.createElement(_checkbox.Checkbox, {
-        name: "enroll-in-research-group",
-        label: "I'd like to participate in product research",
-        onChange: function onChange(event) {
-          return _this2.setState({
-            enrollInResearchGroup: event.target.checked
-          });
-        }
       }))) : _react.default.createElement(_react.Fragment, null)));
     }
   }]);
--- a/node_modules/@atlaskit/feedback-collector/dist/cjs/components/FeedbackForm.js.flow
+++ b/node_modules/@atlaskit/feedback-collector/dist/cjs/components/FeedbackForm.js.flow
@@ -121,18 +121,6 @@ export default class FeedbackForm extends Component<Props, FormFields> {
                   }
                 />
               </Field>
-
-              <Field>
-                <Checkbox
-                  name="enroll-in-research-group"
-                  label="I'd like to participate in product research"
-                  onChange={event =>
-                    this.setState({
-                      enrollInResearchGroup: event.target.checked,
-                    })
-                  }
-                />
-              </Field>
             </Fragment>
           ) : (
             <Fragment />
--- a/node_modules/@atlaskit/feedback-collector/dist/esm/components/FeedbackForm.js
+++ b/node_modules/@atlaskit/feedback-collector/dist/esm/components/FeedbackForm.js
@@ -150,14 +150,6 @@ function (_Component) {
             canBeContacted: event.target.checked
           });
         }
-      })), React.createElement(Field, null, React.createElement(Checkbox, {
-        name: "enroll-in-research-group",
-        label: "I'd like to participate in product research",
-        onChange: function onChange(event) {
-          return _this2.setState({
-            enrollInResearchGroup: event.target.checked
-          });
-        }
       }))) : React.createElement(Fragment, null)));
     }
   }]);
--- a/node_modules/@atlaskit/feedback-collector/dist/esm/components/FeedbackForm.js.flow
+++ b/node_modules/@atlaskit/feedback-collector/dist/esm/components/FeedbackForm.js.flow
@@ -121,18 +121,6 @@ export default class FeedbackForm extends Component<Props, FormFields> {
                   }
                 />
               </Field>
-
-              <Field>
-                <Checkbox
-                  name="enroll-in-research-group"
-                  label="I'd like to participate in product research"
-                  onChange={event =>
-                    this.setState({
-                      enrollInResearchGroup: event.target.checked,
-                    })
-                  }
-                />
-              </Field>
             </Fragment>
           ) : (
             <Fragment />
--- a/node_modules/@atlaskit/feedback-collector/src/components/FeedbackForm.js
+++ b/node_modules/@atlaskit/feedback-collector/src/components/FeedbackForm.js
@@ -121,18 +121,6 @@ export default class FeedbackForm extends Component<Props, FormFields> {
                   }
                 />
               </Field>
-
-              <Field>
-                <Checkbox
-                  name="enroll-in-research-group"
-                  label="I'd like to participate in product research"
-                  onChange={event =>
-                    this.setState({
-                      enrollInResearchGroup: event.target.checked,
-                    })
-                  }
-                />
-              </Field>
             </Fragment>
           ) : (
             <Fragment />
