diff --git a/node_modules/@atlaskit/lozenge/dist/cjs/Lozenge/Content.js b/node_modules/@atlaskit/lozenge/dist/cjs/Lozenge/Content.js
index df6c837..ec3bf4c 100644
--- a/node_modules/@atlaskit/lozenge/dist/cjs/Lozenge/Content.js
+++ b/node_modules/@atlaskit/lozenge/dist/cjs/Lozenge/Content.js
@@ -4,10 +4,12 @@ var core_1 = require("@emotion/core");
 var constants_1 = require("@atlaskit/theme/constants");
 var HORIZONTAL_SPACING = constants_1.gridSize() / 2 + "px";
 exports.default = (function (_a) {
-    var maxWidth = _a.maxWidth, children = _a.children;
-    return (core_1.jsx("span", { css: {
+    var maxWidth = _a.maxWidth, textColor = _a.textColor, children = _a.children;
+    return (core_1.jsx("span", {
+        css: {
             display: 'inline-block',
             verticalAlign: 'top',
+            color: textColor,
             overflow: 'hidden',
             textOverflow: 'ellipsis',
             whiteSpace: 'nowrap',
@@ -15,6 +17,7 @@ exports.default = (function (_a) {
             padding: "0 " + HORIZONTAL_SPACING,
             maxWidth: typeof maxWidth === 'number' ? maxWidth + "px" : maxWidth,
             width: '100%',
-        } }, children));
+        }
+    }, children));
 });
 //# sourceMappingURL=Content.js.map
\ No newline at end of file
diff --git a/node_modules/@atlaskit/lozenge/dist/esm/Lozenge/Content.js b/node_modules/@atlaskit/lozenge/dist/esm/Lozenge/Content.js
index f461e72..616fa56 100644
--- a/node_modules/@atlaskit/lozenge/dist/esm/Lozenge/Content.js
+++ b/node_modules/@atlaskit/lozenge/dist/esm/Lozenge/Content.js
@@ -2,10 +2,12 @@ import { jsx } from '@emotion/core';
 import { gridSize } from '@atlaskit/theme/constants';
 var HORIZONTAL_SPACING = gridSize() / 2 + "px";
 export default (function (_a) {
-    var maxWidth = _a.maxWidth, children = _a.children;
-    return (jsx("span", { css: {
+    var maxWidth = _a.maxWidth, textColor = _a.textColor, children = _a.children;
+    return (jsx("span", {
+        css: {
             display: 'inline-block',
             verticalAlign: 'top',
+            color: textColor,
             overflow: 'hidden',
             textOverflow: 'ellipsis',
             whiteSpace: 'nowrap',
@@ -13,6 +15,7 @@ export default (function (_a) {
             padding: "0 " + HORIZONTAL_SPACING,
             maxWidth: typeof maxWidth === 'number' ? maxWidth + "px" : maxWidth,
             width: '100%',
-        } }, children));
+        }
+    }, children));
 });
 //# sourceMappingURL=Content.js.map
\ No newline at end of file
