diff --git a/node_modules/@types/analytics-node/index.d.ts b/node_modules/@types/analytics-node/index.d.ts
index fba1ffc..5b2722b 100644
--- a/node_modules/@types/analytics-node/index.d.ts
+++ b/node_modules/@types/analytics-node/index.d.ts
@@ -44,6 +44,7 @@ declare namespace AnalyticsNode {
       flushInterval?: number,
       host?: string,
       enable?: boolean,
+      agent?: any,
       timeout?: number | string,
     });
 
